repo/packages/M/marathon-lb/19/marathon.json.mustache

```diff
@@ -9,7 +9,15 @@
     "docker": {
       "image": "{{resource.assets.container.docker.marathon-lb-docker}}",
       "network": "HOST",
-      "privileged": true
+      "privileged": true,
+      "parameters": [
```

Comment:
@nlsun THANK YOU for your patience on this!

I'm properly identifying these parameters as labels (which do properly get attached to the containers when launched).

Really sorry about this!